src/src/com/microsoft/aad/adal/UnexpectedServerResponseException.java

```diff
@@ -0,0 +1,14 @@
+package com.microsoft.aad.adal;
+
+import java.io.IOException;
+
+/**
+ * Created by imakhal on 2/17/2016.
+ */
+public class UnexpectedServerResponseException extends IOException {
```

Comment:
The definition of IOException is "This class is the general class of exceptions produced by failed or interrupted I/O operations."  That's not how UnexpectedServerResponseException is being used above.  It is being used at least in some cases where we simply got values back from the server that the code doesn't know how to handle.  That's not an I/O problem.  I'll have to think about this some more this evening to come up with a better suggestion.